src/stylesheets/components/_buttons.scss

```diff
@@ -111,29 +93,31 @@
   border-radius: 0 4px 4px 0;
 }
 
-/*--- Active styles ---*/
-
-.open > .btn--ghost--inverse {
-  @include button($white, $grey-1, $grey-2);
-}
-
-/*--- Toggle styles ---*/
+/*--- Toggles ---*/
 
 .toggle--menu {
-  > [class*="btn--"] {
-    margin-bottom: 0;
-  }
   .btn--close {
+    display: none;
     float: right;
     @include links($grey-1, none, $grey-2, $grey-1);
     @include milli;
     margin-bottom: 0;
   }
+
+  &.open > .btn--ghost--inverse {
```

Comment:
Would be better to have this line positioned below the declarations of the `btn--ghost--inverse` theme. There are other themes with toggles that are in the buttons view that potentially will have other values. Would be good to identify the pattern for all open buttons.